ReleaseNotes.md

```diff
@@ -1,5 +1,38 @@
 # Release Notes
 
+## New in Release 2.13.0
+
+Post-training Quantization:
+
+- Features:
+  - (OpenVINO) Implemented the use of the Scale Estimation algorithm to calculate quantization parameters within GPTQ for more accurate weight compression. The `gptq` and `scale_estimation` options can now be used together to calculate quantization parameters using the Scale Estimation algorithm within GPTQ in `nncf.compress_weights()`.
+  - (OpenVINO) Added LoRA Correction Algorithm to further improve the accuracy of int4 compressed models on top of other algorithms - AWQ and Scale Estimation. It can be enabled via the optional `lora_correction` parameter of the `nncf.compress_weights()` API. The algorithm increases compression time and incurs a negligible model size overhead. Refer to [accuracy/footprint trade-off](docs/usage/post_training_compression/weights_compression/Usage.md#accuracyfootprint-trade-off) for different int4 compression methods.
+  - (OpenVINO) Added support for combining GPTQ with the AWQ algorithms for more accurate weight compression. The `awq` and `gptq` options can now be used together to sequentially apply AWQ followed by GPTQ in `nncf.compress_weights()`.
+  - (PyTorch) Added implementation of the experimental Post-training Activation Pruning algorithm. Refer to [Activation Sparsity](nncf/experimental/torch/sparsify_activations/ActivationSparsity.md) for details.
+  - Added a memory monitoring tool for logging the memory a piece of python code or a script allocates. Refer to [NNCF tools](tools/README.md) for details.
+- Fixes:
+  - (OpenVINO) Fixed the quantization of Convolution and LSTMSequence operations in cases where some inputs are part of a ShapeOF subgraph.
+  - (OpenVINO) Fixed issue with the FakeConvert duplication for FP8.
+  - Fixed Smooth Quant algorithm issue in case of the incorrect shapes.
+  - Fixed non-deterministic layer-wise scheduling.
+- Improvements:
+  - (OpenVINO) Increased hardware-fused pattern coverage.
+  - Improved progress bar logic during weights compression for more accurate remaining time estimation.
+  - Extended Scale estimation bitness range support for the `nncf.compress_weights()`.
+  - Removed extra logging for the algorithm-generated ignored scope.
+- Tutorials:
+  - [Post-Training Optimization of Flux.1 Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/flux.1-image-generation/flux.1-image-generation.ipynb)
```

Comment:
Please also add the following tutorials with NNCF support (they were added recently and will be highlighted as part of OV 2024.4):
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/internvl2
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/qwen2-audio
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/nuextract-structure-extraction
- https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/minicpm-v-multimodal-chatbot

cc @l-bat @nikita-savelyevv

Reply:
Added